miner/worker.go

```diff
@@ -1039,15 +1050,18 @@ func (w *worker) commit(uncles []*types.Header, interval func(), update bool, st
 	if err != nil {
 		return err
 	}
-	if tr := s.GetTrie(); tr.IsVerkle() {
-		vtr := tr.(*trie.VerkleTrie)
-		// Generate the proof if we are using a verkle tree
-		p, err := vtr.ProveAndSerialize(s.Witness().Keys(), s.Witness().KeyVals())
-		if err != nil {
-			return err
+	fmt.Printf("block %s/%s - %s\n", block.Number(), block.ParentHash().Hex(), w.current.state.GetBalance(block.Coinbase()).String())
+	/*
+		if tr := s.GetTrie(); tr.IsVerkle() {
+			vtr := tr.(*trie.VerkleTrie)
+			// Generate the proof if we are using a verkle tree
+			p, err := vtr.ProveAndSerialize(s.Witness().Keys(), s.Witness().KeyVals())
```

Comment:
there seems to be a problem: no more calls to `ProveAndSerialize` in the codebase...